tf2onnx/graph.py

```diff
@@ -633,6 +641,12 @@ def remove_node(self, node_name):
             if op_output in self._dtypes:
                 del self._dtypes[op_output]
 
+        for op_input in node.input:
+            if op_input not in self._output_to_consumers:
+                raise RuntimeError(
+                    "Input %r of node %r not found." % (op_input, node_name))
```

Comment:
Should this be a make_sure?

Reply:
I hesitated because there was a mixed use of make_sure and RuntimeError in graph.py.